cob_people_detection/ros/launch/face_capture.launch

```diff
@@ -2,13 +2,10 @@
 
 <launch>
   <!-- face capture node (captures faces and stores them in the database) -->
-  <rosparam command="load" ns="/cob_people_detection/face_capture" file="$(find cob_people_detection)/ros/launch/face_recognizer_params.yaml"/>
-  <node name="face_capture" pkg="cob_people_detection" ns="/cob_people_detection/face_capture" type="face_capture_node" output="screen"><!-- launch-prefix="gdb -ex run args"--><!--launch-prefix="valgrind"-->
-    <remap from="face_detections" to="/cob_people_detection/face_detector/face_positions"/>
-    <!--remap from="color_image" to="/cob_people_detection/image_flip/colorimage_out"/--> <!-- only activate on cob3 robots -->
-    <remap from="color_image" to="/cob_people_detection/sensor_message_gateway/colorimage_out"/>
```

Comment:
Has the remapping of the incoming color image been removed? Please leave it there with the right namespace otherwise it is impossible to train new people. Or did you solve this in another way?